acra/configuring-maintaining/general-configuration/acra-addzone.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: acra-addzone
+title: AcraAddzone
```

Comment:
The reason why I changed this, because I saw that in `General configuration` block we have sections with different titles.

Is it ok to have such different naming that some of them are in camel case but others in CLI based format? So my question would be why Keymaker is represented in camel case like `AcraKeymaker`  but not like `acra-keymaker` so I decided to unify all these section to be in camel case. 

![Screenshot from 2021-09-15 09-47-12](https://user-images.githubusercontent.com/24659475/133384486-15524b9e-e82d-428c-8c17-b1868448b7ac.png)

Reply:
Discussed in slack. Lets follow cli format representation for all cli tools.